submission/main.go

```diff
@@ -0,0 +1,121 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package submission
```

Comment:
I recommend putting this executable into a separate package, e.g. `submission/(ct_)?proxy`. One reason is that operators (including us) might want to redefine `main.go` to plug in internal transport/monitoring/etc systems, so it would be optimal to not bake the unused stuff into `submission` package.

Reply:
Another name: `submission/server`

Reply:
Done